### DISCUSSION TITLE
Project Alive?

### DISCUSSION THREAD

**Discussion by matthoskins1980** [Ladder99/fanuc-driver#117] [General]
Hello!  I recently found this project and am using it to start talking to my fanuc controlled machines.  Just curious - is this project alive?  It's been a while since there's been activity - but that can also be a sign of code maturity - so I thought I'd ask here.  Thanks!

**Comment by dheepaklab** [comment]
Hi, have you done it? I'm trying to do this with my Raspberry Pi.
